cli/plugins/builder/cmd.go

```diff
@@ -0,0 +1,155 @@
+package builder
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/project-flogo/cli/common"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+
+	fePluginCommand.Flags().StringVar(&file, "file", "", "Path to Flogo application json file")
+	fePluginCommand.Flags().StringVar(&file, "f", "", "Path to Flogo application json file")
+
+	fePluginCommand.Flags().StringVar(&name, "name", "", "Name of the binary. If not provided, binary with name <APPNAME>-<OS_NAME>-<OS_ARCH> will be generated")
+	fePluginCommand.Flags().StringVar(&name, "n", "", "Name of the binary. If not provided, binary with name <APPNAME>-<OS_NAME>-<OS_ARCH> will be generated")
+
+	fePluginCommand.Flags().StringVar(&output, "output", "", "Folder where binary to be created. By default, binary will be created in current directory")
+	fePluginCommand.Flags().StringVar(&output, "o", "", "Folder where binary to be created. By default, binary will be created in current directory")
+
+	fePluginCommand.Flags().StringVar(&platform, "platform", "", "Build binary for specific platform. Value must be specified in the form of <OS_NAME>/<OS_ARCHITECTURE> e.g. linux/386, linux/amd64, By default, current OS is used")
```

Comment:
We should also support ```GOOS``` and ```GOARCH``` environment vars.

Reply:
It supports by native,  Go Build will take GOOS and GOARCH env var while building the binary

Reply:
This has been fixed at FE builder side